src/main.h

```diff
@@ -237,7 +241,7 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, bool fRejectAbsurdFee=false);
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, CAmount nAbsurdFee=0);
```

Comment:
@jtimon made it `const` (pass by reference does not make sense for ints).

Not sure about making it mandatory.